src/libostree/ostree-repo.c

```diff
@@ -4911,6 +4911,96 @@ ostree_repo_verify_commit_ext (OstreeRepo    *self,
 }
 
 /**
+ * ostree_repo_verify_data:
```

Comment:
Can we name this `ostree_repo_gpg_verify_data` or something for a bit of extra clarity?